src/AppInstallerCLITests/Command.cpp

```diff
@@ -554,3 +566,51 @@ TEST_CASE("ParseArguments_UnknownName", "[command]")
 
     REQUIRE_COMMAND_EXCEPTION(command.ParseArguments(inv, args), CLI::Resource::String::InvalidNameError(Utility::LocIndView{ values[1] }));
 }
+
+TEST_CASE("ParseArguments_PositionalWithMultipleValues", "[command]")
+{
+    Args args;
+    TestCommand command({
+            Argument{ "multi", 'm', Args::Type::Query, DefaultDesc, ArgumentType::Positional }.SetCountLimit(5),

```

Comment:
For clarity and consistency, might be better to use MultiQuery type in the tests. I got a bit confused when I saw just `Query` as I thought it was supposed to be for just a single package